<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1050</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Polynomials</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/img/icon.ico</normaloff>:/img/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="dbListWidget">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>100</y>
      <width>501</width>
      <height>801</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>1091</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Operations</string>
    </property>
    <widget class="QLineEdit" name="inputField1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>14</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputField2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextBrowser" name="outputResult">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>130</y>
       <width>571</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>18</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>161</width>
       <height>251</height>
      </rect>
     </property>
     <property name="title">
      <string>Operators</string>
     </property>
     <widget class="QRadioButton" name="rbtnPlus">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnMinus">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnDerivative">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>f(x)⁽ⁿ⁾</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnMultiply">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnDivision">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnEqual">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>150</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>==</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnPoint">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>82</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>f(x)=</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btnGetResult">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Get Result</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bntClearInputField2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>100</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bntClearInputField1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bntClearOutputResult">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>100</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSave">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>200</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Save to Database</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>130</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>50</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Trebuchet MS</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Database</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>100</y>
      <width>761</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Integer roots</string>
    </property>
    <widget class="QLineEdit" name="inputField3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>130</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>№</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bntClearInputField3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>100</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>-&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextBrowser" name="outputResult2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>461</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFindRoots">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Find roots</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bntClearOutputField2">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>30</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>720</y>
      <width>801</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Input</string>
    </property>
    <widget class="QPushButton" name="bntClearInputPolymon">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputPolynom">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>59</y>
       <width>601</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
       <pointsize>16</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QPushButton" name="btnAddPolynom">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>60</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add polynom</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
